spec/fixtures/output/k8s/pod_yml_no_mounts.yml

```diff
@@ -47,9 +47,6 @@ spec:
     - "spec"
     ports:
     - containerPort: 8080
-    volumeMounts:
-    - name: configmap-volume
-      mountPath: /ood
```

Comment:
This test file/test case was added #239 as a bugfix. You can see in the yaml it still has the volume, though we get rid of the volume mount.

Maybe we need an additional test case where this handles "no volumes or mounts whatsoever" and another that handles "no volumes but we do mount a configmap".

Reply:
I made this particular test case go back to previous behavior and added new test case with no mounts and no configmap mounts.